all-packages.nix

```diff
@@ -15,5 +20,6 @@
   };
 
   callPackage = newScope (self // nixpkgs-candidates // {inherit callPackage;});
+  php-callPackage = php-newScope (self // nixpkgs-candidates // {inherit callPackage;});
```

Comment:
Because packaging `flarum` required us to point to a different fork of `nixpkgs`, we defined a separate `callPackage` that uses that fork